Source/Core/Plugins/EventParser/Raygun/RaygunEventParserPlugin.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+using Exceptionless.Core.Pipeline;
+
+namespace Exceptionless.Core.Plugins.EventParser.Raygun {
+    [Priority(0)]
+    public class RaygunEventParserPlugin : IEventParserPlugin {
+        private readonly RaygunEventMapper _mapper = new RaygunEventMapper();
+
+        public List<PersistentEvent> ParseEvents(string input, int apiVersion, string userAgent) {
+            if (userAgent != "raygun" && apiVersion != 1)
+                return null;
+            
+            var events = new List<PersistentEvent>();
+            switch (input.GetJsonType()) {
+                case JsonType.Object: {
```

Comment:
Ah, I guess I never paid that close attention. I think I just automatically do it for multi line big blocks of code, it's easier to tell where it starts and ends.